package.json

```diff
@@ -68,12 +69,16 @@
     "karma-webpack": "1.7.0",
     "lodash": "4.3.0",
     "mocha": "2.4.5",
-    "react": "15.2.1",
+    "react": "15.5.4",
     "react-addons-test-utils": "15.0.0",
-    "react-dom": "15.0.0",
+    "react-create-class": "1.0.0",
```

Comment:
Looks like this package isn't necessary (apparently they switched names to `create-react-class`, which you have above)